x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -283,6 +283,10 @@ export class TaskRunner<
     const ruleType = this.ruleTypeRegistry.get(ruleTypeId);
 
     const ruleLabel = `${this.ruleType.id}:${ruleId}: '${name}'`;
+    const loggerId = rule.alertTypeId.startsWith('.')
```

Comment:
The `rule.alertTypeId` is the same as `this.ruleType.id`, which is available in the constructor. WDYT of initializing the logger in the constructor using `this.ruleType.id` and then we can use the same logger everywhere in this task runner?

Reply:
Yes that sounds good! Sorry I didn't realize that is what you were asking earlier!

Reply:
Resolved in https://github.com/elastic/kibana/pull/142121/commits/a6646cdb79b885aef72e59441f2bb15ac940fd81